### PR TITLE
feat: remove faucet on mainnet

### PR DESCRIPTION
**Pull request by sighWang**

### CODE REVIEW

**Review by zhangyouxin — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -25,7 +25,6 @@ function App() {
                 <Page>
                   <PageHeader onViewChange={handleViewChange}></PageHeader>
                   <LightGodwokenApp activeView={activeView} />
-                  <PageFooter></PageFooter>
```
Comment (zhangyouxin) [inline src/App.tsx:28]: Remove import of `PageFooter` also

File: src/components/Layout/PageHeader.tsx [hunk 2]
```diff
@@ -92,17 +92,7 @@ const PageHeader: React.FC<Props> = ({ onViewChange }) => {
...
-          content={() => <PopoverMenu handleClick={closePopoverMenu}></PopoverMenu>}
-          trigger="click"
-          overlayClassName="popover-menu"
-          visible={popoverVisible}
-          placement="bottomLeft"
-        >
-          <Hamburger className="hamburger-menu" onClick={openPopoverMenu}></Hamburger>
-        </Popover>
-      </div>
+      <div className="right-side"></div>
```
Comment (zhangyouxin) [inline src/components/Layout/PageHeader.tsx:95]: Is it necessary to leave a blank div here?

Comment (sighWang) [reply src/components/Layout/PageHeader.tsx:95]: yes, this div have width and make layout correct

**Comment by sighWang** [comment]
close this pr cause main branch code should keep merge able to develop branch

**Pull request closed without merging**